src/spacetimedb.ts

```diff
@@ -397,18 +333,15 @@ export class SpacetimeDBClient {
    * @param table The table to unsubscribe from
    */
   public removeManualTable(table: string) {
+    // pgoldman 2024-06-25: Is this broken? `registerManualTable` treats `manualTableSubscriptions`
```

Comment:
Honestly not sure, but it looks like it. It was added for the website stuff. @PuruVJ do you know if this is even used? if not then maybe let's remove

Reply:
We don't use the SDK in the website at all, so if that's the only place its used, its safe to remove